components/molecule/accordion/src/Tab/index.js

```diff
@@ -0,0 +1,112 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+const BASE_CLASS = 'sui-MoleculeAccordion-tab'
+const CONTENT_CLASS = `${BASE_CLASS}Content`
+const CONTENT_CONTAINER_CLASS = `${CONTENT_CLASS}Container`
+const CONTAINER_BUTTON_CLASS = `${BASE_CLASS}TitleContainer`
+const OPEN_CLASS = 'is-open'
+const BUTTON_CLASS = `${BASE_CLASS}Btn`
+const BUTTON_CONTENT_CLASS = `${BUTTON_CLASS}Content`
+const BUTTON_TITLE_CLASS = `${BUTTON_CLASS}Title`
+const ICON_CLASS = `${BASE_CLASS}Icon`
+const MAX_HEIGHT = 100
+
+class Tab extends Component {
```

Comment:
I think this component could be easily move to function and could help with readability. Not a must, but maybe is worth it.